components/Facts.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+	<ul class="container">
+		<li v-for="fact in facts" :key="fact.stat" class="fact">
```

Comment:
I don't think the `fact.stat` is a proper `key`. There could be statistics which say "40% = male, 40% = female, 20% = other". In that case you'll have a `key` of `40%` twice